Examples/HelloWorld/HelloWorld-simple/HelloWorld-simple.cs

```diff
@@ -0,0 +1,73 @@
+﻿//  ------------------------------------------------------------------------------------
+//  Copyright (c) Chuck Rolke
```

Comment:
//  Copyright (c) Microsoft Corporation

Microsoft OSS projects require the above Copyright line. Without that we cannot accept the contribution.